src/app/containers/FrontPageSection/index.jsx

```diff
@@ -111,6 +139,7 @@ const FrontPageSection = ({ bar, group, sectionNumber }) => {
     // (<section> tags *should* imply `role="region"`)
     // While this may be true in a perfect world, we set it in order to get
     // the greatest possible support.
+    /* eslint-disable no-nested-ternary */
```

Comment:
is this needed? if so, we probably shouldn't disable a rule like this for an entire file

Reply:
Fixed it, now it's only disabled for one line. But I cannot remove it fully because `eslint` starts complaining